W 13763167 bandaid ping doc duke (#81)
mulesoft/docs-code-builder: modules/ROOT/pages/ping-locally-deployed-app.adoc

//LOGO (web, desktop, or both)
include::partial$acb-ide-logos.adoc[tags="both-ides"]

> Because Anypoint Code Builder works as a web IDE, consider that you must ping a different URL to trigger the execution of locally deployed Mule applications.

Author: I find it odd/somewhat disorienting that we call a web-based deployment that we run in the IDE a "local" deployment. I think that terminology might confuse people because on the web, the deployment (when you build the app in the IDE) is hosted _remotely_.

Is "local" common terminology for this situation? If not, I think we should consider using some other term. 

Alternatively, perhaps we can think of it as a local development environment or something? See below for a stab at this approach.

Still, I don't know what people will make of the page title: "Ping a Locally Deployed App"

Also, is the term "ping" commonly used the way we're using it? When I look it up in technical docs on the web, I don't think the definition aligns precisely with what we're doing here when we send the URL via a browser or REST client to trigger the flow. Any suggestions for an alternative word or corrections to my understanding of "ping"?

Author: And any suggestion for an alternative title?

> When an integration that you are developing builds successfully, the application deploys _locally_, within the IDE. It is common to configure an HTTP Listener to trigger the execution of a deployed application through a URL.

Author: ```suggestion
When an integration that you are developing builds successfully, the application deploys _locally_, within your development environment, whether on your desktop or to the web-based host for your Anypoint Code Builder applications. It is common to configure an HTTP Listener to trigger the execution of a deployed application through a URL.
```

> When an integration that you are developing builds successfully, the application deploys _locally_, within your development environment, whether on your desktop or to the web-based host for your Anypoint Code Builder applications. It is common to configure an HTTP Listener to trigger the execution of a deployed application through a URL.

Author: ```suggestion
When an integration that you are developing builds successfully, the application deploys _locally_, within your development environment, whether on your desktop or to the web-based host for your Anypoint Code Builder applications. It is common to configure an HTTP Listener to trigger the execution of a deployed application through a URL. The URLs to use for the desktop and web IDEs differ:
```

> When an integration that you are developing builds successfully, the application deploys _locally_ within your development environment, whether on your desktop or to the web-based host for your Anypoint Code Builder applications. It is common to configure an HTTP Listener to trigger the execution of a deployed application through a URL. The URLs to use for the desktop and web IDEs differ:

Author: ```suggestion
When an integration that you are developing builds successfully, the application deploys _locally_ within your development environment, whether on your desktop or from the web-based host for your Anypoint Code Builder applications. It is common to configure an HTTP Listener to trigger the execution of a deployed application through a URL. The URLs to use for the desktop and web IDEs differ:
```

> When an integration that you are developing builds successfully, the application deploys _locally_, within your development environment, whether on your desktop or to the web-based host for your Anypoint Code Builder applications. It is common to configure an HTTP Listener to trigger the execution of a deployed application through a URL. The URLs to use for the desktop and web IDEs differ:

Author: ```suggestion
When an integration that you are developing builds successfully, the application deploys _locally_ within your development environment, whether on your desktop or to the web-based host for your Anypoint Code Builder applications. It is common to configure an HTTP Listener to trigger the execution of a deployed application through a URL. The URLs to use for the desktop and web IDEs differ:
```

> When an integration that you are developing builds successfully, the application deploys _locally_ within your development environment, whether on your desktop or from the web-based host for your Anypoint Code Builder applications. It is common to configure an HTTP Listener to trigger the execution of a deployed application through a URL. The URLs to use for the desktop and web IDEs differ:

Author: ```suggestion
When an integration that you are developing builds successfully, the application deploys _locally_ within your development environment, whether on your desktop or from the web-based host for your Anypoint Code Builder applications. 

It is common to configure an HTTP Listener to trigger the execution of a deployed application through a URL. The URLs to use for the desktop and web IDEs differ:
```

* In the desktop IDE, a URL configured for the listener, such as `localhost:8081`, works if no other applications are using that port. 

> * In the web IDE, you must use a proxied URL to trigger your application. 

Author: ```suggestion
* In the web IDE, you must use a URL to a proxy that triggers your application. 
```

> * In the web IDE, you must use a URL to a proxy that triggers your application. 

Author: ```suggestion
* In the web IDE, your application is hosted remotely, so you must configure a URL to a proxy for your application's external host.
```

> * In the web IDE, your deployed application is hosted remotely, so you must configure a URL to a proxy for your application's external host.

Author: ```suggestion
* In the web IDE, the local deployment within the IDE is hosted remotely, so you must configure a URL to a proxy for your application's external host. 
```

> * In the web IDE, the local deployment within the IDE is hosted remotely, so you must configure a URL to a proxy for your application's external host. 

Author: ```suggestion
* In the web IDE, your local development environment is hosted remotely, so you must configure a URL to a proxy for your application's external host. 
```

//LOGO (web, desktop, or both)
include::partial$acb-ide-logos.adoc[tags="both-ides"]

> When an integration that you are developing builds successfully, the application deploys _locally_ within your development environment, whether on your desktop or from the web-based host for your Anypoint Code Builder applications. 

Author: ```suggestion
When an integration that you are developing builds successfully, the application deploys _locally_ within your development environment, whether on your desktop or the web-based host for your Anypoint Code Builder applications. 
```

> It is common to configure an HTTP Listener to trigger the execution of a deployed application through a URL. The URLs to use for the desktop and web IDEs differ:

Author: ```suggestion
It is common to configure an HTTP Listener to trigger the execution of a deployed application through a URL. The URLs to use for the desktop and web IDEs differ for deployments to your development environment:
```

* In the desktop IDE, a URL configured for the listener, such as `localhost:8081`, works if no other applications are using that port. 

> * In the web IDE, your local development environment is hosted remotely, so you must configure a URL to a proxy for your application's external host. 

Author: ```suggestion
* In the web IDE, your local development environment is hosted remotely, so you must configure a URL to a proxy for your application's host. 
```
